### ISSUE TITLE
'X Trackers Blocked' is a link. It should be a button. ⛓

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#256]
'11 Trackers Blocked' is a link. It should be a button.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43591590-ab59174a-9641-11e8-889b-c14cdb9f78c4.png">
